docker_signal.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
After doing some reading, it seems `SIGTERM` is actually the most appropriate signal to send for cleanup. See https://www.quora.com/What-is-the-difference-between-the-SIGINT-and-SIGTERM-signals-in-Linux and https://en.wikipedia.org/wiki/Unix_signal.

This means the worker probably needs to catch SIGTERM and gracefully shutdown.